### DISCUSSION TITLE
removal of 'html_login'

### DISCUSSION THREAD

**Discussion by Tuumke** [sabnzbd/sabnzbd#2248] [General]
I couldn't reply in the other discussion. But i don't use it since i have [Authelia ](https://github.com/authelia/authelia) handle my authentication for me

**Comment by thezoggy** [comment]
dupe of https://github.com/sabnzbd/sabnzbd/discussions/2197

**Comment by Safihre** [comment]
Indeed, and as mention there:
Based on all your feedback, the html_login will be kept. Thanks all!
